CMakeLists.txt

```diff
@@ -846,6 +847,7 @@ include(cmake/OpenCVGenHeaders.cmake)
 
 # Generate opencv.pc for pkg-config command
 if(NOT OPENCV_SKIP_PKGCONFIG_GENERATION
```

Comment:
Can we remove `OPENCV_SKIP_PKGCONFIG_GENERATION`?

Reply:
It is still here for compatibility reasons.
However `OPENCV_GENERATE_PKGCONFIG` is disabled by default in master, so use case with enabling both options looks strange.

Perhaps we should maintain somehow list of deprecated build options, warn about them, and drop in the future.